addons/stock/models/stock_traceability.py

```diff
@@ -92,8 +92,7 @@ def get_lines(self, line_id=None, **kw):
                     ('lot_id', '=', context.get('active_id')),
                     ('location_id.usage', '!=', 'internal'),
                     ('state', '=', 'done'),
-                    ('move_id.returned_move_ids', '=', False),
```

Comment:
Is not faster a domain using “move_id != False” as now you are expecting a valid move_id